openmc/data/data.py

```diff
@@ -1,10 +1,9 @@
 import itertools
+from math import sqrt
```

Comment:
Out of curiosity, why do you prefer `math.sqrt` over `numpy.sqrt`?

Reply:
The `sqrt` is only used for scalar quantities. `numpy.sqrt` is appropriate if you're actually going to take the square root of a vector, but if you're only using it for a scalar, it adds overhead compared to `math.sqrt`.